source/Halibut.Tests/Support/Streams/NoSyncIOStream.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+using Halibut.Transport.Streams;
+
+namespace Halibut.Tests.Support.Streams
+{
+    public class NoSyncIoStream : DelegateStreamBase
```

Comment:
Can this describe what it actually does? It doesn't do 'NoSyncIo', it just logs when sync IO occurs